recipes/medpy/meta.yaml

```diff
@@ -8,19 +8,27 @@ package:
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: f8a94937dbb947ab069e767862dc6b86896b153c41ce8ed9369c7d79c0033a88
+  patches:
+    - graphcut_boost_lib.patch
 
 build:
-  noarch: python
-  number: 0
+  number: 1
   script:
+    - export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
     - export CXXFLAGS="${CXXFLAGS} -I${PREFIX}/include -I${PYTHON_INCLUDE_DIR}"
     - "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+    - cd $( find ${PREFIX}/lib/python*/site-packages/medpy/ -type d -name graphcut ) && find . -name maxflow.cython*.so -exec ln -sr {} maxflow.so \;
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
   host:
+    - boost
```

Comment:
can we use boost-cpp here, or is boost needed?

Reply:
The patch indicates that the python library is indeed needed.